x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/editor/formula_help.tsx

```diff
@@ -391,6 +391,14 @@ export function getFunctionSignatureLabel(
         defaultMessage: '[kql]?: string, [lucene]?: string',
       });
     }
+    if (def.filterable && def.shiftable) {
+      extraArgs += ', ';
+    }
+    if (def.shiftable) {
+      extraArgs += i18n.translate('xpack.lens.formula.shiftExtraArguments', {
+        defaultMessage: '[shift]?: string',
+      });
+    }
```

Comment:
The logic here breaks when transitioning to a shifted count when using the `en-xa` locale, it produces `count(Recordsshift='3h')` instead of `count(shift='3h')`. Maybe we're doing some string equality?